src/index.ts

```diff
@@ -224,6 +224,28 @@ function b32decodeXemAddr(data: string): Buffer {
   return nemSdk.default.utils.convert.ua2hex(nemSdk.default.model.address.b32decode(address));
 }
 
+function ontHexToBase58(hexEncoded: Buffer): string {
```

Comment:
These aren't hex, they're binary - can you change the names accordingly?

Reply:
I changed the name of the functions to ontBase58CheckDecoder and ontBase58CheckEncoder